### ISSUE TITLE
[feature request] conda-forge package

### ISSUE THREAD

**Issue by cdiener** [jotech/gapseq#60]
The installation for gapseq is a bit painful and requires root access to the machine which is often not the case for servers. It would be awesome if there was a conda package to facilitate a more accessible installation.

**Comment by jotech** [comment]
Hi @cdiener thanks for pointing this out! We hope to have a conda package available soon!
Right now, I added some hints to the [readme](https://github.com/jotech/gapseq/blob/master/docs/install.md) how to install all needed dependencies from conda without root rights.

**Comment by cdiener** [comment]
Or do you have a working docker or singularity image (without the DBs)?

**Comment by jotech** [comment]
good idea thanks! we try to come up with some alternative installation in the near future :)

**Comment by jotech** [comment]
just as intermediate solution, we added a [conda installation tutorial](https://github.com/jotech/gapseq/blob/master/docs/install.md#conda)

**Comment by cdiener** [comment]
Sure, no hurries. We packaged it as a docker image for our workflows for now: https://github.com/Gibbons-Lab/gapseq-docker. Though I noticed there are some dependencies that are not mentioned, like curl for instance.

**Comment by nick-youngblut** [comment]
@jotech any updates on creating the bioconda recipe? As illustrated with your [conda installation tutorial](https://github.com/jotech/gapseq/blob/master/docs/install.md#conda), a conda recipe really helps developers (eg., of bioinformatics pipelines) install & maintain the dependencies of all software.

**Comment by nick-youngblut** [comment]
In regards to the conda install tutorial, a bit simpler approach:

conda env yaml file:

```
channels:
- conda-forge
- bioconda
dependencies:
- bash
- r
- perl
- parallel
- gawk
- sed
- grep
- bc
- git
- coreutils
- wget
- barrnap
- bedtools
- exonerate
- glpk
- hmmer
- blast
- bioconda::libsbml 
- r-data.table
- r-stringr
- r-stringi
- r-getopt
- r-doParallel
- r-foreach
- r-r.utils
- r-sybil
- r-biocmanager
- bioconductor-biostrings
- r-jsonlite
- r-renv
```

Create conda env:

```
mamba env create -n gapseq -f gapseq_env.yaml
```

Additional R package install:

```
R -e 'renv::install(c("glpkAPI", "CHNOSZ"))'
```


In regards to the gapseq install instructions, I'm guessing that you meant:

```
bash ./src/update_sequences.sh
```

instead of:

```
src/./update_sequences.sh
```

**Comment by Waschina** [comment]
Thanks @nick-youngblut. I agree that a conda env yaml file would clean up the instructions. I'll update the conda instructions in January.

**Comment by jolespin** [comment]
Once this is available as a conda installation I'm going to add it as a module in my https://github.com/jolespin/veba pipeline.  The publication just got accepted and should be out soon.

**Comment by cmkobel** [comment]
@jotech in order for a bioconda package to be effective you should make frequent releases (maybe, at least once a year). The current release is more than two years old even though you had commits a few weeks ago (https://github.com/jotech/gapseq/commit/2d739a56a26a1eb9841b0b4999b3498452923406). Are there any plans to make a new release of gapseq that includes the changes to the development version?
